docs/badge.md

```diff
@@ -57,6 +64,9 @@ import { Avatar, Badge } from 'react-native-elements'
     containerStyle={{ position: 'absolute', top: -4, right: -4 }}
   />
 </View>
+
+const BadgedIcon = withBadge(1)(Icon)
+<BadgedIcon type="ionicon" name="ios-chatbubbles" />
 ```
 
```

Comment:
Could you just add the example with the `connect` thing we talked above ? I think it's a pretty common use case